plugin/api.go

```diff
@@ -283,6 +283,12 @@ type API interface {
 	// Minimum server version: 6.2
 	UpdateUserCustomStatus(userID string, customStatus *model.CustomStatus) *model.AppError
 
+	// RestoreToPreviousCustomStatus will reset user's custom status to a previously custom status set by user itself.
+	//
+	// @tag User
+	// Minimum server version: 7.2
+	RestoreToPreviousCustomStatus(userID string) *model.AppError
```

Comment:
Note that this will need to be updated if this PR is not merged with 7.2 (I think that's obvious though)